src/main/kotlin/graphics/scenery/controls/behaviours/MouseDrag.kt

```diff
@@ -0,0 +1,67 @@
+package graphics.scenery.controls.behaviours
+
+import graphics.scenery.BoundingGrid
+import graphics.scenery.Camera
+import graphics.scenery.Node
+import graphics.scenery.utils.LazyLogger
+import graphics.scenery.utils.extensions.plus
+import graphics.scenery.utils.extensions.times
+import org.scijava.ui.behaviour.DragBehaviour
+import kotlin.reflect.KProperty
+
+/**
+ * Drag nodes roughly along the view plane axis by mouse.
+ * Implements algorithm from https://forum.unity.com/threads/implement-a-drag-and-drop-script-with-c.130515/
+ *
+ * @author Jan Tiemann <j.tiemann@hzdr.de>
+ */
+open class MouseDrag(
```

Comment:
sciview already has some dragging behaviours that @kephale implemented (of which I don't remember the details :-D) -- did you check these?

Reply:
Did not but did now. 

Sceneview algo is more exact. It moves the node along the viewplane. But the cursor and object move at different perceived speeds.
(This) Scenery algo is not exact. It moves along a sphere around the camera but looks nicer because cursor and node move together.

What do you say @skalarproduktraum @kephale is more important?

Reply:
Ported behaviors from sciView. Once this PR is merged and available in sciView we can delete the behaviors there.